apps/deploy.html.md.erb

```diff
@@ -0,0 +1,61 @@
+---
+title: Deploy a Release
+layout: docs
+nav: firecracker
+titlecase: false
+order: 30
+---
+
+<%= partial "/docs/partials/v2_transition_banner" %>
+
+`fly deploy` creates a new release of your Fly App and updates the app's Machines as a group.
+
+<div class="callout">
+**Note:** The first time you run `fly deploy` on an older Machines App&mdash;an app created using `fly create --machines` or `fly machine run`&mdash;it will migrate that app to the Fly Apps Platform and unify the configuration for all the app's existing Machines. **Do not run `fly deploy` on a pre-V2 app with existing Fly Machines, if they have different configs and/or code bases and you want to keep it that way.** See [**Migrate a Machines App to Apps V2**](/docs/apps/migrate-to-v2) for more information.
```

Comment:
very minor inspection in the callout at line 13:

"older Machines App" is also "pre-V2 app" right? If so, the different terms used for the same thing makes me question whether they point to the same thing or not. I feel it might be better to use just one term instead

Reply:
👍 changed